tests/index.js

```diff
@@ -23,6 +23,20 @@ describe('all rule files should be exported by the plugin', function() {
   });
 });
 
+describe('deprecated rules', function() {
+  it('marks all deprecated rules as deprecated', function() {
```

Comment:
I wonder if we should just use the `deprecated` property to dynamically build up the `deprecatedRules` list?